ui/src/queries/posts.ts

```diff
@@ -0,0 +1,47 @@
+import { useQuery, UseQueryOptions, UseQueryResult } from "react-query";
+import { apiRequest } from "../utils/apiRequest";
+
+export interface Post {
+  id: number;
+  author: string;
+  authorId: string;
+  description: string;
+  skillsPossessed: string[]; // TODO: constants
```

Comment:
Should we make the `Skills` list passed in by config so that we don't need to keep the API and UI in sync, or shall we live with it for a while?